docs/api/skills/abstract_round_abci/utils.md

```diff
@@ -16,7 +16,7 @@ Tool to verify Randomness retrieved from various external APIs.
 
 The ciphersuite used is BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_NUL_
 
-https://drand.love/docs/specification/`cryptographic`-specification
```

Comment:
These two drand links contain a #, and pydocs-markdown removes it and reformats the link, effectively breaking it. I have not found a way to escape this behaviour or replace the hashtag with an equivalent code. We could use a url shortener but i felt like this way is more informative.